alpharaw/match/spec_finder.py

```diff
@@ -9,8 +11,31 @@ def find_spec_idxes_by_rt(
     query_stop_rt: float,
     query_left_mz: float,
     query_right_mz: float,
-):
+) -> np.ndarray:
+    """
+    Find MS2 spectrum indices (int32) from the `spectrum_df`
```

Comment:
maybe comment (here or somewhere else) why int32 is set explicitly here

Reply:
Done